src/main.d.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Object whose keys are signal names and values are signal objects.
+ */
+export declare const signalsByName: { [signalName: string]: Signal }
+/**
+ * Object whose keys are signal numbers and values are signal objects.
+ */
+export declare const signalsByNumber: { [signalNumber: number]: Signal }
+
+export declare type SignalAction =
+  | 'terminate'
+  | 'core'
+  | 'ignore'
+  | 'pause'
+  | 'unpause'
+export declare type SignalStandard =
+  | 'ansi'
+  | 'posix'
+  | 'bsd'
+  | 'systemv'
+  | 'other'
+
+export declare type Signal = {
+  /**
+   * Standard name of the signal, for example 'SIGINT'.
+   */
+  name: string
+  /**
+   * Code number of the signal, for example 2. While most number are cross-platform, some are different between different OS.
+   */
+  number: number
+  /**
+   * Human-friendly description for the signal, for example 'User interruption with CTRL-C'.
+   */
+  /** this is the description description */
```

Comment:
This comment `this is the description description` should be removed?

Reply:
Oo nice catch!